pkg/Makefile.am

```diff
@@ -13,6 +13,19 @@ playground_DATA = \
 dist/playground/extra.de.po: pkg/playground/extra.de.po
 	$(COPY_RULE)
 
+machinesdir = $(pkgdatadir)/machines
+
+machines_novnc_symlinks:
+	find bower_components/no-vnc/include/ -regex ".*\.\(js\|css\|ttf\|woff\)" -a ! -name '*keysymdef.js' -exec ln -srf '{}' dist/machines/include/ \;
+
+machines_DATA = \
+	dist/machines/include/keysymdef.js \
+	machines_novnc_symlinks \
```

Comment:
automake rules are not that magical - i. e. accepting a phony rule name here, instead of plain file names:
```
$ make install DESTDIR=`pwd`/i
 /usr/bin/install -c -m 644 dist/machines/include/keysymdef.js ./machines_novnc_symlinks '/home/martin/upstream/cockpit/i/usr/share/cockpit/machines'
/usr/bin/install: cannot stat './machines_novnc_symlinks': No such file or directory
```

This needs to be spelled out like this rather: `machines_DATA = [...] $(shell find ...)`.